src/main/java/com/transferwise/openbanking/client/oauth/PrivateKeyJwtAuthentication.java

```diff
@@ -37,7 +37,7 @@ private String getClientAssertionToken(AspspDetails aspspDetails) {
         JwtClaims jwtClaims = new JwtClaims();
         jwtClaims.setIssuer(aspspDetails.getClientId());
         jwtClaims.setSubject(aspspDetails.getClientId());
-        jwtClaims.setAudience(aspspDetails.getPrivateKeyJwtAuthenticationAudience());
+        jwtClaims.setAudience(aspspDetails.getPrivateKeyJwtAuthenticationAudience() !=null ? aspspDetails.getPrivateKeyJwtAuthenticationAudience() : aspspDetails.getTokenUrl());
```

Comment:
implement a default method in AspDetails which will return the getToken(). Instead of having null checks